src/ingestion.rs

```diff
@@ -100,16 +100,20 @@ fn setup_ingestion_metrics(metrics: &Registry) -> IngestionMetrics {
 }
 
 fn create_ingestion_counter(
-    metrics: &Registry,
+    metrics: Option<&Registry>,
```

Comment:
Why? Is there a reason tests cannot include counters?

Reply:
Starting and stopping docker seems to create an error with prometheus, i.e. it doesn't seem to allow parallel registry. Instead of investigating it I just removed it for tests since metrics are not needed in it. If we wanted to test it, then ok, maybe in the future we could try to reintroduce them.